pkg/controller/migplan/registry.go

```diff
@@ -97,6 +111,11 @@ func (r ReconcileMigPlan) ensureRegistrySecret(client k8sclient.Client, plan *mi
 		return nil, err
```

Comment:
Seems like we missed adding `liberr.Wrap` to the some of the returns in this method. Would you mind adding it?